main/src/main/java/org/mobilitydata/gtfsvalidator/cli/Main.java

```diff
@@ -49,7 +49,14 @@
   public static void main(String[] argv) {
     Arguments args = new Arguments();
     CliParametersAnalyzer cliParametersAnalyzer = new CliParametersAnalyzer();
-    new JCommander(args).parse(argv);
+    JCommander jCommander = new JCommander(args);
+    jCommander.parse(argv);
+    if (args.getHelp()) {
+      printHelp(jCommander);
+      System.out.println(
+          "⚠️ Note that parameters marked with an asterisk (*) in the help menu are mandatory.");
+      return;
+    }
     if (!cliParametersAnalyzer.isValid(args)) {
```

Comment:
While you are here: please move creation of CliParametersAnalyzer to that line.

if (!new CliParametersAnalyzer().isValid(args)) {
}

Just curious: why CliParametersAnalyzer.isValid is not static?

Reply:
> While you are here: please move creation of CliParametersAnalyzer to that line.

👍🏾

> Just curious: why CliParametersAnalyzer.isValid is not static?

It actually should be, will make the change in a separate PR. Thanks!

Reply:
See #893.